docs/src/templates/pages/styleguide/ui-components/listings.hbs

```diff
@@ -9,12 +9,32 @@ navactive: listings
 ---
 <p>A listing of items.  Visually split by separators</p>
 
-<h3>Fully linked Listing Items</h3>
-<p>Listing in which each `list–item` is a link (so that the whole item is clickable).</p>
+<h2 class="sg-sectionHeading">Fully linked Listing Items with Address</h2>
+
 {{#>demo-block baseUrl=../baseUrl }}
-<div class="c-card c-card--noPadVertical u-bordered c-listing">
+<div class="c-listing">
```

Comment:
Would be good to have this as a listing component that you pass params to (to add the extra classname for offline and for the type of listing module to pull in).

(see @DamianMullins last PR for a way to do this)